### PR TITLE
Loading data

### PR DESCRIPTION
**Pull request by JoannaNawara**
We created the environment, script to download data, load data and script for showing maps with points.

### CODE REVIEW

File: data/Wojewodztwa.zip
Comment (jstaszel-agh) [inline data/Wojewodztwa.zip:1]: If you want to keep bigger files within repository, you can use git lfs.

File: environment.yml
Comment (jstaszel-agh) [inline environment.yml:1]: Please start using conda-lock and also if you use conda environment, why do you have packages added with pip?

File: src/plot_map.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import pandas as pd
+import geopandas as gpd
+import matplotlib.pyplot as plt 
+from src.data_ingestion import create_path_to_data
```
Comment (jstaszel-agh) [inline src/plot_map.py]: When you are inside a package (src), you can use relative paths (from .data_ingestion import ...)

**Review by jstaszel-agh — COMMENTED** [review]
Please update environment and resolve other comments. If something is unclear or you do not agree, just let me know and we can discuss.
You also have __init__.py in your main directory, that shouldn't be there.
image

**Comment by JoannaNawara** [comment]
@jstaszel-agh Corrections are added. Can you review and accept the merge, please? :)

**Comment by JoannaNawara** [comment]
I will merge this as changes where really small, so I think everything is correct

**Pull request merged**